core/src/main/java/com/kodebeagle/javaparser/MethodInvocationResolver.java

```diff
@@ -50,6 +54,7 @@
     protected Map<String, String> typeTosuperType = new HashMap<>();
     private Map<String, List<Object>> typeToInterfacesFullyQualifiedName = new HashMap<>();
     protected Map<String, List<String>> typeToInterfaces = new HashMap<>();
+    public Map<String, TypeJavadoc> typeJavadocs = new HashMap<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make typeJavadocs a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)